Userland/Libraries/LibGfx/ImageFormats/BilevelImage.h

```diff
@@ -35,6 +35,17 @@ class BilevelImage {
     void set_bit(size_t x, size_t y, bool b);
     void fill(bool b);
 
+    // This is binary compatible with JBIG2::CombinationOperator.
```

Comment:
nit: i'd add a `blend_type_from_combination_operator(op) { switch(op) { case CO::Or: return BT::Or; …]` in the jbig2 code instead of static_casting and requiring matching enum values